python/python/lance/dataset.py

```diff
@@ -223,13 +223,25 @@ def join(
 
     def versions(self):
         """
-        Return all versions in this dataset
+        Return all versions in this dataset.
         """
         versions = self._ds.versions()
         for v in versions:
-            v["timestamp"] = datetime.fromtimestamp(v["timestamp"])
+            # TODO: python datetime supports only microsecond precision. When a separate Version object is
+            # implemented, expose the precise timestamp (ns) to python.
+            ts_nanos = v["timestamp"]
+            v["timestamp"] = datetime.fromtimestamp(ts_nanos // 1e9) + timedelta(
+                microseconds=(ts_nanos % 1e9) // 1e3
+            )
         return versions
 
+    @property
+    def active_version(self) -> int:
```

Comment:
Just one nit. 

Could you change this name to `version` as property to be backward compatible with 0.2 version

https://github.com/eto-ai/lance/blob/archive/python/lance/_lib.pyx#L330?

Reply:
Done.